packages/sdk/client/src/packlets/client/service-definitions.ts

```diff
@@ -59,6 +62,7 @@ export const clientServiceBundle = createServiceBundle<ClientServices>({
   SpacesService: schema.getService('dxos.client.services.SpacesService'),
   DataService: schema.getService('dxos.echo.service.DataService'),
   NetworkService: schema.getService('dxos.client.services.NetworkService'),
+  LoggingService: schema.getService('dxos.client.services.LoggingService'),
```

Comment:
I think we can remove the tracing service entirely, and rely on TRACE level logs to cover its functionality